apps/crn-server/src/data-providers/contentful/team.data-provider.ts

```diff
@@ -71,9 +72,6 @@ export class TeamContentfulDataProvider implements TeamDataProvider {
             ...searchTerms.map((term) => ({
               projectTitle_contains: term,
             })),
-            {
```

Comment:
We currently list teams fetching from the API. I think if we remove this query, it will break the search by tags
<img width="1436" alt="Screenshot 2024-02-27 at 06 38 00" src="https://github.com/yldio/asap-hub/assets/16595804/3b9bf032-4902-4a48-b998-6659835ebd11">

Reply:
interesting. If searching by tags after we add this relationship is difficult (or impossible) we might want to consider adding a follow up ticket to move this search to algolia